### PR TITLE
Containers: new build system based on containers

### PR DESCRIPTION
**Pull request by mrpollo**
documents how to use the new docker containers to build QGC apps
directly without having to install the requirements on your local
development machine.

### CODE REVIEW

**Comment by mrpollo** [comment]
cc @LorenzMeier @dagar @DonLakeFlyer

File: en/SUMMARY.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Summary
 
 * [Overview](README.md)
-* [Getting Started](getting_started/README.md)
+* [Getting Started with source & builds)](getting_started/README.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:4]: FWIW I wouldn't have changed this, but it does no harm

**Review by hamishwillee — APPROVED** [review]
Looks good to me.

**Comment by DonLakeFlyer** [comment]
This will be fantastic to prevent screw ups. I spend a significant amount of time dealing with problems which ultimately relate back to folks simply not following the instructions for dependencies. In the dev guide for build I might even be more strong about the container based building being the "recommended" way of doing this. The more we push people to this the less screw up there will be.

**Pull request merged**